yoti-sdk-api/src/main/java/com/yoti/api/client/docs/session/create/check/RequestedFaceComparisonCheck.java

```diff
@@ -0,0 +1,74 @@
+package com.yoti.api.client.docs.session.create.check;
+
+import static com.yoti.api.client.spi.remote.util.Validation.notNullOrEmpty;
+
+import com.yoti.api.client.docs.DocScanConstants;
+
+/**
+ * Requests creation of a FaceComparisonCheck
+ */
+public class RequestedFaceComparisonCheck extends RequestedCheck<RequestedFaceComparisonConfig> {
+
+    private final RequestedFaceComparisonConfig config;
+
+    RequestedFaceComparisonCheck(RequestedFaceComparisonConfig config) {
+        this.config = config;
+    }
+
+    public static RequestedFaceComparisonCheck.Builder builder() {
+        return new RequestedFaceComparisonCheck.Builder();
+    }
+
+    @Override
+    public String getType() {
+        return DocScanConstants.FACE_COMPARISON;
+    }
+
+    @Override
+    public RequestedFaceComparisonConfig getConfig() {
+        return config;
+    }
+
+    public static class Builder {
```

Comment:
`private` constructor on the builder class please

Reply:
If do that, it will be impossible to call public methods on the `builder` itself as we do not allow to hold a reference to this class(as it _private_)

Reply:
I got it wrong. Sorry, I thought about the class itself
Added private constructor